tools/importer/scripts/roo-tales-article-thumbnail-scraper.js

```diff
@@ -0,0 +1,236 @@
+// Description: AJAX-based scraper for Qantas Newsroom roo tales articles metadata
+// and associated thumbnail images
+
+/* eslint-disable no-console, import/no-unresolved */
+
+const axios = require('axios');
+const fs = require('fs');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const FormData = require('form-data');
+const { setTimeout } = require('timers/promises');
+
+// Configuration
+const config = {
+  // AJAX endpoint
+  ajaxUrl: 'https://www.qantasnewsroom.com.au/wp-admin/admin-ajax.php',
+
+  // Base URL for constructing article URLs
+  baseUrl: 'https://www.qantasnewsroom.com.au/roo-tales/',
+
+  // Page range to scrape
+  pageRange: {
+    start: 1,
+    end: 19,
+  },
+
+  // Headers for the request
+  headers: {
+    accept: 'application/json, text/javascript, */*',
+    'content-type': 'application/x-www-form-urlencoded; charset=UTF-8',
+    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36',
+  },
+
+  // Form data template
+  formData: {
+    action: 'wp_loadmore',
+    // page_no will be set dynamically
+  },
+
+  // Rate limiting and retry settings
+  rateLimit: {
+    delay: 1000, // Delay between requests in milliseconds
+    maxRetries: 3, // Maximum number of retries for failed requests
+    retryDelay: 2000, // Delay between retries in milliseconds
+  },
+};
+
+class RooTalesArticleThumbnailScraper {
+  // eslint-disable-next-line no-shadow
+  constructor(config) {
+    this.config = config;
+    this.articleMap = {}; // To store article-image pairs
```

Comment:
```suggestion
    this.articleMap = new Map(); // To store article-image pairs
```

Minor: `articleMap` object can be replaced with Map for better performance with large datasets.